site/_docs/reference.md

```diff
@@ -2654,11 +2654,14 @@ BigQuery's type system uses confusingly different names for types and functions:
 | s | ARRAY(expr [, expr ]*)                         | Construct an array in Apache Spark
 | s | ARRAY_COMPACT(array)                           | Removes null values from the *array*
 | b | ARRAY_CONCAT(array [, array ]*)                | Concatenates one or more arrays. If any input argument is `NULL` the function returns `NULL`
-| s | ARRAY_DISTINCT(array)                          | Returns unique elements of *array*. Keeps ordering of elements.
+| s | ARRAY_DISTINCT(array)                          | Removes duplicate values from the *array*. Keeps ordering of elements
```

Comment:
In fact, `Keeps ordering of elements` is a sentence that I'm worried about.

This sentence lacks a subject, I hope the description is close to natural language.
For example:
`Removes duplicate values from the *array* that keeps ordering of elements`